Documentation/ceph-fs-mirror-crd.md

```diff
@@ -39,8 +39,61 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 ### FilesystemMirror Settings
 
+* `peers`: to configure mirroring peers
+  * `secretNames`:  a list of peers to connect to. Currently (Ceph Pacific release) **only a single** peer is supported where a peer represents a Ceph cluster.
+  However, if you want to enable mirroring of multiple filesystems, you would have to have **one Secret per filesystem**.
 * `placement`: The cephfs-mirror pods can be given standard Kubernetes placement restrictions with `nodeAffinity`, `tolerations`, `podAffinity`, and `podAntiAffinity` similar to placement defined for daemons configured by the [cluster CRD](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/cluster.yaml).
 * `annotations`: Key value pair list of annotations to add.
 * `labels`: Key value pair list of labels to add.
 * `resources`: The resource requirements for the cephfs-mirror pods.
 * `priorityClassName`: The priority class to set on the cephfs-mirror pods.
+
+### Configuring mirroring peers
+
+On an external site you want to mirror with, you need to create a bootstrap peer token.
+The token will be used by one site to **pull** images from the other site.
+The following assumes the name of the pool is "test" and the site name "europe" (just like the region), so we will be pulling images from this site:
+
+```console
+external-cluster-console # ceph fs snapshot mirror peer_bootstrap create myfs2 client.mirror europe
+{"token": "eyJmc2lkIjogIjgyYjdlZDkyLTczYjAtNGIyMi1hOGI3LWVkOTQ4M2UyODc1NiIsICJmaWxlc3lzdGVtIjogIm15ZnMyIiwgInVzZXIiOiAiY2xpZW50Lm1pcnJvciIsICJzaXRlX25hbWUiOiAidGVzdCIsICJrZXkiOiAiQVFEVVAxSmdqM3RYQVJBQWs1cEU4cDI1ZUhld2lQK0ZXRm9uOVE9PSIsICJtb25faG9zdCI6ICJbdjI6MTAuOTYuMTQyLjIxMzozMzAwLHYxOjEwLjk2LjE0Mi4yMTM6Njc4OV0sW3YyOjEwLjk2LjIxNy4yMDc6MzMwMCx2MToxMC45Ni4yMTcuMjA3OjY3ODldLFt2MjoxMC45OS4xMC4xNTc6MzMwMCx2MToxMC45OS4xMC4xNTc6Njc4OV0ifQ=="}
```

Comment:
Seems like the doc doesn't need to include a long example token
```suggestion
{"token": "<token>"}
```

Reply:
I still think it's good for users to see the token resembles to the one in the example.